@bazel/webpack/bin/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+# This target is used locally by the repository to access webpack-worker.
+# directory is designed to match the userland external repository so that
+# we don't have to do label substitution in release packages.
+# this target represent the webpack-worker entry in bin section of package.
+# This is a npm_package_bin target in userland repository.
+# NOTE: this file is not meant to be released in the npm package. be careful
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+
+nodejs_binary(
+    name = "webpack-worker",
+    data = [
+        "@npm//@bazel/worker",
+        "@npm//webpack-cli",
```

Comment:
Since webpack is a peer dependency of the cli, I think you might need to also include it as a data dependency here

Reply:
Good catch!